In [12]:
import numpy as np

# Assignment 1


**Problem 1** - 3 pts each:  Write the solution to the following indefinite integrals using LaTeX.  No credit will be awarded for work turned in in any other format.  Follow the model below.
$$
\int \sin(x) dx = -\cos(x) + C
$$

<ol>
<li>    $$
        \int \cos^{2}(4x) dx
        $$
    
<li>
    $$
    \int \frac{dt}{1+9t^2}
    $$
<li>
    $$
    \int y^{n} dy
    $$

_Answers to Problem 1_: Write your answers in the lines below using LaTeX.

1.1:   
    $$
   \int \cos^{2}(4x) dx  = \frac{x}{2} + \frac {\sin(8x)}{16} + C
    $$

1.2:
    $$
    \int \frac{dt}{1+9t^2} = \frac{1}{3} \arctan(3t) + C
    $$

1.3:
    $$
    \int y^{n} dy = \frac {y^ {n+1}}{n+1}
    $$
    

**Problem 2** - 3 pts each: Using the classic result for a geometric series that 

$$
\sum_{n=0}^{\infty}x^{n} = \frac{1}{1-x}, ~ \left|x\right|<1,
$$

find the following sums.  Use the above statement of the sum of a geometric series as a model.  Again, use LaTeX to write your answers.  No credit will be given otherwise. 

<ol>

<li> 
$$
\sum_{j=0}^{\infty} 2^{j/3}x^{5j}, ~ |x|<\frac{1}{2^{1/15}}.
$$

<li>
$$
\sum_{j=0}^{\infty} \cos^{3j}(x), ~ 0 < |x| < \pi.
$$

<li>
$$
\sum_{j=2}^{\infty} e^{2jx}, ~ x<0. ~ \mbox{(PAY ATTENTION TO THE LOWER LIMIT ON THE SUM)}
$$


_Answers to Problem 2_: Write your answers below using LaTeX.  Again, use the beginning of the problem statement of Problem 2 as a model for your answers.  

2.1: 
$$
\sum_{j=0}^{\infty} 2^{j/3}x^{5j}= \frac{1}{1-2^{1/3}x^{5}}
$$

2.2:
$$
\sum_{j=0}^{\infty} \cos^{3j}(x) = \frac{1}{1-\cos^{3}(x)}
$$

2.3:
$$
\sum_{j=2}^{\infty} e^{2jx} = \frac{e^{4x}}{1-e^{2x}}
$$

**Problem 3** 10 pts: So, if you go ask the right kind of people, they will tell you that you can find $\pi$ via the following limit. 

$$
\pi = \lim_{n\rightarrow\infty} \frac{4}{n^{2}}\sum_{k=0}^{n}\sqrt{n^{2}-k^{2}}
$$

So if I give you the following Python function 

In [13]:
def pi_approx(n):
    tot = 0.
    for kk in range(0,int(n)+1):
        tot += np.sqrt(n**2.-kk**2.)
    return 4.*tot/n**2.

using a list and a for loop as in the lecture, determine $\pi$ to 7-correct digits.    

In [ ]:
values = [1e5, 1e6, 1e7, 1e8, 1e9]

for value in values:
    print( "pi   = %1.7f"  %pi_approx(value)) 

pi   = 3.1416126
pi   = 3.1415947
pi   = 3.1415929
pi   = 3.1415927


_Provide a justification for your answer using the results from your code here._

$$
\pi = 3.14159265359 \approx 3.1415927
$$
Based off Google Calculator!